api/envoy/extensions/access_loggers/grpc/v3/als.proto

```diff
@@ -91,10 +91,9 @@ message CommonGrpcAccessLogConfig {
   repeated string filter_state_objects_to_log = 5;
 
   // Sets the retry policy when the establishment of a gRPC stream fails.
-  // If the stream succeeds once in establishing If the stream succeeds
-  // at least once in establishing itself, no retry will be performed
-  // no matter what gRPC status is received. Note that only
-  // :ref:`num_retries <envoy_v3_api_field_config.core.v3.RetryPolicy.num_retries>`
+  // If the stream succeeds at least once in establishing in establishing itself,
```

Comment:
```suggestion
  // If the stream succeeds at least once in establishing itself,
```

Reply:
Hmm.. Sorry. This happened second time yesterday. Some keyboard magic looks like